kratos/python_scripts/gid_output_process.py

```diff
@@ -645,3 +653,48 @@ def __remove_list_files(self):
                         os.remove(f)
                     except WindowsError:
                         pass
+                    
+    #
+    def __remove_post_results_files(self, step_label):
```

Comment:
Well, I deliberately used double leading underscore for internal stuff (things that, if I was in C, I would declare as private functions). If you prefer to avoid name mangling, this is something I would be open to discuss.

I agree that this kind of _magical_ python stuff should not be used without a good reason, however.